protocol/xmrmaker/claim.go

```diff
@@ -38,12 +40,13 @@ func (s *swapState) claimFunds() (ethcommon.Hash, error) {
 		}
 	}
 
+	ethBalance, err := s.ETHClient().Balance(s.ctx)
```

Comment:
I would call this `weiBalance` to avoid confusion.  Or you could convert it into an ETH balance out here instead of just in the log on line 49. One of the improvements that `apd.Decimal` provides over `big.Int` is an `IsZero` function.